projects/aws/upgrader/upgrade/kubeadm.go

```diff
@@ -57,35 +65,55 @@ func (u *InPlaceUpgrader) KubeAdmInFirstCP(ctx context.Context) error {
 		}
 	}
 
-	if err = u.appendKubeletConfig(ctx, newKubeAdmConfig); err != nil {
-		return fmt.Errorf("appending kubelet config: %v", err)
-	}
-
 	if err = u.backUpAndDeleteCoreDNSConfig(ctx, componentsDir); err != nil {
 		return fmt.Errorf("backing up and deleting coreDNS config: %v", err)
 	}
 
-	kubeAdmVersionCmd := []string{"kubeadm", "version"}
+	kubeAdmVersionCmd := []string{"kubeadm", "version", "-oshort"}
 	version, err := u.ExecCommand(ctx, kubeAdmVersionCmd[0], kubeAdmVersionCmd[1:]...)
 	if err != nil {
 		return execError(kubeAdmVersionCmd, string(version))
 	}
-	logger.Info("current version of kubeadm", "cmd", "kubeadm version", "output", string(version))
+	logger.Info("current version of kubeadm", "cmd", "kubeadm version -oshort", "output", string(version))
 
-	kubeAdmUpgPlanCmd := []string{"kubeadm", "upgrade", "plan", "--ignore-preflight-errors=CoreDNSUnsupportedPlugins,CoreDNSMigration", "--config", newKubeAdmConfig}
```

Comment:
Are we not running upgrade plan anymore?

Reply:
i just moved around stuff cos i didn't have to append kubelet config for 1.30 plus versions. so both of these run upgrade plan but at different flow. just some reordering i guess.